build_local_windows.ps1

```diff
@@ -52,7 +52,15 @@ python -m venv "$tmpdir/dcos_build_venv"
 . "$tmpdir/dcos_build_venv/Scripts/Activate.ps1"
 
 # Install the DC/OS tools
-./prep_local_windows.ps1
-
+#./prep_local_windows.ps1
+./prep_teamcity_windows.ps1
 # Build a release of DC/OS
 release create $env:USERNAME local_build windows
+# Build tar ball for windows. 2 params: packages location and DC/OS variant:
+./build_genconf_windows.ps1 "$HOME\dcos-artifacts\windows"
+## Set AWS Credentials:
+# Set-AWSCredential -ProfileName teamcity-server
```

Comment:
@pcrews and @orsenthil can you please have a look at this?

Reply:
Sure. - This is powershell semantics. We could try it and see if this works.
But the logic sounds good to me.  Why is this commented out ?

Reply:
because I am not sure if Profile name is correct. please specify whatever is suitable on the build agents

Reply:
so can we uncomment it? @orsenthil can you provide the profile name?

Reply:
Hi Sergii - Please use environment variables

- AWS_ACCESS_KEY_ID
- AWS_SECRET_ACCESS_KEY

Instead of Profile. We don't provide the profile, but expose via environment variables.
- The region will be us-west-2

The artifacts could be uploaded to this location (example)
```
https://s3-us-west-2.amazonaws.com/downloads.dcos.io/dcos/testing/master/commit/1e7a71f3a2f5229b66367ae3bd98ed4ced3e72f8/dcos_generate_config.ee.sh
```

Reply:
HI @sergiimatusEPAM  - You could get this to upload to "any" location in `downloads.dcos.io/dcos/testing/master/` bucket. And once it is successful,  we could replace this with a variable that will controlled via Teamcity parameters  (You could leave the location as any hardcoded one for this commit, which will be replaced next by us).